pupil-api/src/config.ts

```diff
@@ -19,9 +24,14 @@ export default {
       Key: process.env.APPINSIGHTS_INSTRUMENTATIONKEY
     }
   },
-  Cors: {
-    Whitelist: process.env.CORS_WHITELIST || ''
+  Redis: {
+    Host: process.env.REDIS_HOST || 'localhost',
+    Port: process.env.REDIS_PORT || 6379,
+    Key: process.env.REDIS_KEY,
+    useTLS: getEnvironment() === 'Local-Dev' ? false : true
   },
-  Environment: getEnvironment(),
-  AzureStorageConnectionString: process.env.AZURE_STORAGE_CONNECTION_STRING
+  RateLimit: {
+    Threshold: process.env.RATE_LIMIT_THRESHOLD || 100,
```

Comment:
any particular reason why 100?
i wonder how this will affect travis test runs 🤔

Reply:
Seems reasonable for larger schools